scripts/e2e/run-card.sh

```
@@ -0,0 +1,98 @@
+#   scripts/e2e/run-card.sh --list
+#
+# 环境变量：
+#   E2E_REMOTE_HOST   远端主机别名 (默认 fedora，需要在 ~/.ssh/config 配好)
+#   E2E_REMOTE_PATH   远端项目路径 (默认 projects/uniclipboard，相对 $HOME)
+#   E2E_SKIP_SYNC=1   跳过 rsync，直接用远端已有代码
+#   E2E_KEEP_PROFILE=1 透传给 wdio.conf.mjs，跑完不清 profile data
+
+set -uo pipefail
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)